README.md

```diff
@@ -0,0 +1,40 @@
+# saturn-client
+Python library for interacting with [Saturn Cloud](https://www.saturncloud.io/) API.
+
+## Connect
+
+```python
+from saturn import SaturnConnection
+
+# From outside of Saturn
+conn = SaturnConnection(url="<SATURN_URL>", api_token="<API_TOKEN>")
```

Comment:
how do I get this URl and API token? I think it's a frustrating experience to start working on something and hit docs like this that have placeholders but no information on how to find good values for them. Could you add that type of information?

Reply:
Yeah. I put it in the docstring, but forgot to put it in the README.